jax/_src/scipy/stats/gumbel_r.py

```diff
@@ -265,5 +255,5 @@ def logsf(x: ArrayLike, loc: ArrayLike = 0, scale: ArrayLike = 1) -> Array:
   neg_z = lax.div(lax.sub(loc, x), scale)
   t1 = lax.neg(lax.exp(neg_z))
   # numerical stability
-  log_sf = _log1mexp(t1)
+  log_sf = log1mexp(-t1)
```

Comment:
nit: remove the `-` here as well as the `lax.neg` above (or possibly just inline the whole thing: the `t1` variable and the `# numerical stability` comment are not all that helpful)